ports/rp2/boards/ARDUINO_NANO_RP2040_CONNECT/mpconfigboard.h

```diff
@@ -7,6 +7,14 @@
 // Enable networking and sockets.
 #define MICROPY_PY_NETWORK              (1)
 #define MICROPY_PY_USOCKET              (1)
+#define MICROPY_PY_WEBREPL              (1)
+#define MICROPY_PY_UWEBSOCKET           (1)
+#define MICROPY_PY_UHASHLIB_SHA1        (1)
+#define MICROPY_PY_OS_DUPTERM           (1)
+#define mp_builtin_open mp_vfs_open
```

Comment:
if you rebase on latest master, this `mp_builtin_open` define shouldn't be needed